include/zephyr/bluetooth/bluetooth.h

```diff
@@ -470,13 +518,16 @@ int bt_id_delete(uint8_t id);
 /**
  * @brief Bluetooth data.
  *
- * Description of different data types that can be encoded into
- * advertising data. Used to form arrays that are passed to the
- * bt_le_adv_start() function.
+ * @details Description of different AD Types that can be encoded into advertising data. Used to
+ * form arrays that are passed to the @ref bt_le_adv_start() function. The @ref BT_DATA define can
```

Comment:
```suggestion
 * form arrays that are passed to the bt_le_adv_start() function. The @ref BT_DATA define can
```
or
```suggestion
 * form arrays that are passed to the @ref bt_le_adv_start function. The @ref BT_DATA define can
```

The `()` adds an implicit reference already